### DISCUSSION TITLE
Documentation Support for MSM8953

### DISCUSSION THREAD

**Discussion by SRIDHARDV** [msm8953-mainline/linux#154] [General]
Hello Community,
I have recently seen that Quectel SC60 is based on MSM8953, hope this documentation may helpful for the developers.
Like camera support, display drivers android boot...,
https://www.quectel.com/ProductDownload/SC60.html

**Comment by bzy-080408** [comment]
This is mainline Linux,not Android or downstream Linux kernel.Camera support is not full by mainline.
For quectel sc60, please visit quectel website and ask it.until you want to running mainline Linux.
